core-web/libs/portlets/edit-ema/portlet/src/lib/components/dot-ema-dialog/dot-ema-dialog.component.ts

```diff
@@ -346,7 +373,15 @@ export class DotEmaDialogComponent {
             }
         });
 
-        this.action.emit({ event: customEvent, payload: this.dialogState().payload });
+        const { payload, dirty, saved, isTranslation } = this.dialogState();
+
+        this.action.emit({
+            event: customEvent,
+            payload,
+            dirty,
```

Comment:
Isn't it easier to emit a status?
So, if you have to create others or delete them, you don't have to keep modifying the emit.

Reply:
Omit my comment.

Reply:
You are right! I like that approach, good catch!